app/models/event_attendance.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: event_attendances
+#
+#  id         :bigint(8)        not null, primary key
+#  user_id    :integer          not null
+#  event_id   :integer          not null
+#  intention  :string           not null
```

Comment:
I see "Yes" and "No" as the values for the intention. Are there likely to be other values? Would an enum be more useful here?

Reply:
Likely there will be at least 1 more intention, something like "Interested".  But finitely many and an enum would still make sense.

Reply:
Created https://github.com/openbrian/osm-microcosms/issues/8 to track this.